docs/src/main/sphinx/release/release-394.md

```diff
@@ -49,6 +49,11 @@
 
 ## Iceberg connector
 
+```{warning}
+This release has a performance regression when reading Parquet data. It is fixed
```

Comment:
It turns out the regression is not widespread, so no need for the warning (see additional discussion in the comment thread for that item)